packages/webapp/__tests__/SquadFeedPage.tsx

```diff
@@ -231,15 +231,52 @@ Object.assign(navigator, {
 
 describe('squad header bar', () => {
   it('should show five of the squad member images', async () => {
-    const members = generateMembersResult().sourceMembers.edges;
+    Object.defineProperty(global, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation(() => ({
+        matches: true,
+        addListener: jest.fn(),
+        removeListener: jest.fn(),
+      })),
+    });
+    const members = generateMembersResult().sourceMembers.edges.slice(0, 5);
+    renderComponent();
+    const lists = await screen.findAllByLabelText('Members list');
+    const list = lists.find(
+      (item) => !item.className.includes('laptopL:hidden'),
+    );
+    expect(list).toBeInTheDocument();
+    const result = await Promise.all(
+      members.map(async ({ node: { user } }) => {
+        const elements = await screen.findAllByAltText(
+          `${user.username}'s profile`,
+        );
+        return expect(elements.length).toEqual(2);
+      }),
+    );
+    const COUNTER_ELEMENT = 1;
+    expect(result.length).toEqual(5);
+    expect(list.childNodes.length).toEqual(result.length + COUNTER_ELEMENT);
+    Object.defineProperty(global, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation(() => ({
+        matches: false,
+        addListener: jest.fn(),
+        removeListener: jest.fn(),
+      })),
+    });
```

Comment:
This seems not needed right?

Reply:
If I don't revert state to how it is by default the next test will fail.

Reply:
Hmm i see ok lets leave for now, ideally it rather be cleaned in global beforeEach in case test order run different or sync